Feature/initial template

osmanyce opened the pull request.

osmanyce committed:
change template location

osmanyce committed:
Merge branch 'dev' into feature/initial-template

osmanyce committed:
test

osmanyce committed:
Merge branch 'dev' of https://github.com/editoblez/fireman into dev

osmanyce committed:
Add role converter. Add users db functions

editoblez commented on src/main/java/com/ec/fireman/beans/RequirementListBean.java:
```diff
@@ -23,9 +26,13 @@
 
 	private static final Logger LOG = LogManager.getLogger(RequirementListBean.class);
 
+	// TODO: IMPORT DAO
```
Este TODO creo que debe ser eliminado debido a que ya estas haciendo el @Inject. Si te refieres al dao de requierement entonces está bien porque aún no lo he puesto. Luego de esta revisión lo pongo

editoblez commented on src/main/java/com/ec/fireman/beans/UserListBean.java:
```diff
@@ -22,13 +24,16 @@
 	private static final long serialVersionUID = -5468228478359216158L;
 
 	private static final Logger LOG = LogManager.getLogger(UserListBean.class);
```
En este código mejor eliminar esta línea y poner sobre la clase `@Log4j2`, para eso tenemos lombok como dependencia, él genera este código por ti.

editoblez commented on src/main/java/com/ec/fireman/beans/UserListBean.java:
```diff
@@ -43,32 +48,35 @@ public void refreshUsers() {
-		LOG.info("Password: " + passReset1 + " - " + passReset2);
+	@Transactional
+	public String resetPassword() {
+		if (passReset1 != passReset2) {
```
Deberíamos usar aquí equals ya que son Strings

editoblez commented on src/main/java/com/ec/fireman/beans/UserListBean.java:
```diff
@@ -43,32 +48,35 @@ public void refreshUsers() {
 		LOG.info("Users length: " + (users != null ? users.size() : 0));
```
Los loggings deberíamos usarlos con la variable `log` debido que es el nombre por defecto que usamos con la anotación de lombok

editoblez commented on src/main/java/com/ec/fireman/beans/UserListBean.java:
```diff
@@ -111,4 +111,12 @@ public void setPassReset2(String passReset2) {
+	}
+
+	public void setRole(Role role) {
+		this.role = role;
```
Configura en tu eclipse con la siguiente configuración en 
Preference -> Java -> Code Style -> Formatter -> Edit
Tab policy : Space Only
Indentation Size: 2
Tab size: 2

De todos modos creo que lo mejor sería que te importaras esta config:
[formatter.xml.zip](https://github.com/editoblez/fireman/files/3313730/formatter.xml.zip)
Tienes que descomprimir estar archivo, dentro hay archivo que se llama formatter.xml, este archivo ya tiene algunas configuraciones que nos ayudaran a estar más concistentes con el formato, siempre hay algunos detalles que vamos a tener que ir puliendo poco a poco con el trabajo. Lo inluyes en esta vista:
![image](https://user-images.githubusercontent.com/6070777/59911453-8fb99000-93d9-11e9-88a3-336c93ded8fb.png)
Presionando **Import**

Te puse este comentario aquí para que no se me olvide, jejjee, pero esto va para todo porque veo que al parecer tienes indentation 4

editoblez commented on src/main/java/com/ec/fireman/data/entities/Role.java:
```diff
@@ -6,32 +6,32 @@
-  @Id
-  @GeneratedValue(strategy = GenerationType.AUTO)
-  private long id;
+	@Id
```
Mira en estas líneas lo que te decía, te sale el cambio porque tenemos indentation diferentes, la idea es no tener tanto.

editoblez commented on src/main/webapp/app/pages/admin/requirementList.xhtml:
```diff
@@ -69,13 +69,32 @@
 									<p:message for="txtDes" />
 								</div>
 							</h:panelGroup>
+
```
Uffff, importa el archivo del formatter pero con todo vamos a revisar cómo te salen los XMLs files, ya que como ves está como le dan la gana

editoblez reviewed (changes requested):
Ya el PR fue revisado, lo más grave e importante es tener un formateador tanto para los xml como los java igual, para los cambios sean consistentes

osmanyce committed:
Merge branch 'dev' of https://github.com/editoblez/fireman into dev

osmanyce committed:
# WARNING: head commit changed in the meantime

Merge branch 'dev' into feature/initial-template

# Conflicts:
#	src/main/webapp/app/pages/admin/userList.xhtml
